bin/carthage.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
This script pins Carthage to a particular version and handled caching of the `/Carthage` directory on CircleCI.